src/gui/speedlimitdlg.cpp

```diff
@@ -74,7 +74,7 @@ void SpeedLimitDialog::updateSpinValue(int val) const
     if (val <= 0) {
         m_ui->spinBandwidth->setValue(0);
         m_ui->spinBandwidth->setSpecialValueText(QString::fromUtf8(C_INFINITY));
-        m_ui->spinBandwidth->setSuffix(QString::fromUtf8(""));
+        m_ui->spinBandwidth->setSuffix({});
```

Comment:
IMO, `""` is better for indicating empty strings. Default QString constructor isn't too much faster.

Reply:
QLatin1String isn't also required for constructing empty strings since there is no encoding.